src/ripples/focus-ripple.jsx

```diff
@@ -57,6 +80,8 @@ const FocusRipple = React.createClass({
   },
 
   _getRippleElement(props) {
+    const theme = this.state.muiTheme.ripple;
```

Comment:
It's already available as a property `this.props.muiTheme`. Can you remove the state logic?